MODULE.bazel

```diff
@@ -88,23 +94,33 @@ use_repo(
     "org_golang_x_sync",
 )
 
-bazel_lib_toolchains = use_extension("@aspect_bazel_lib//lib:extensions.bzl", "toolchains")
-use_repo(
-    bazel_lib_toolchains,
-    "copy_directory_toolchains",
-    "jq_toolchains",
-)
+#######
+# OCI #
+#######
 
-register_toolchains(
-    "@copy_directory_toolchains//:all",
-    "@jq_toolchains//:all",
+oci = use_extension("@rules_oci//oci:extensions.bzl", "oci")
+oci.pull(
```

Comment:
Could we re-add the comments for WORSPACE how we checked the version. I'd like to turn this into a github action at some point (bump, run tests and send PR).